akka-contrib/src/test/scala/akka/contrib/throttle/TimerBasedThrottlerSpec.scala

```diff
@@ -78,15 +78,15 @@ class TimerBasedThrottlerSpec extends TestKit(ActorSystem("TimerBasedThrottlerSp
       throttler ! SetTarget(Some(echo))
       1 to 3 foreach { throttler ! _ }
       throttler ! SetTarget(None)
-      within(1 second) {
-        1 to 3 foreach { expectMsg(_) }
+      within(0.34 second) {
```

Comment:
This might be too tight for our CI environment, long GC pauses happen rather frequently. Is it possible to adjust the test to allow for a longer duration here?